### PR TITLE
[MAR-2807]

### PR DESCRIPTION
**Pull request by anthonymckerron**
Adding functions/html for the adding/removing and displaying for RFXs.

### CODE REVIEW

File: app/templates/view_buyers.html [hunk 1]
```diff
@@ -81,35 +84,97 @@
   {% endcall %}
 
   <br>
-{% if brief %}
+{% if brief%}
 
-    <b> Edit brief data</b>
+{# Add user to the brief #}
```
Comment (malmas18) [inline app/templates/view_buyers.html]: as discussed, please remove these comments

File: app/main/views/buyers.py [hunk 2]
```diff
@@ -113,6 +113,17 @@ def update_brief(brief_id):
...
                                   },
                        'update_details': {'updated_by': current_user.email_address}}).get('briefs')
+        elif request.form.get('add_seller_to_rfx_brief'):
+            brief = (data_api_client.req.briefs(brief_id)
+                                        .suppliers(request.form['add_seller_to_rfx_brief'].strip())
+                                        .put({'update_details': {
+                                              'updated_by': current_user.email_address}}).get('briefs'))
+        elif request.form.get('remove_seller_from_rfx_brief'):
+            brief = (data_api_client.req.briefs(brief_id).suppliers(
+                    request.form['remove_seller_from_rfx_brief'].strip())
```
Comment (malmas18) [inline app/main/views/buyers.py]: let me go through the formatting here with you

File: app/main/views/buyers.py [hunk 3]
```diff
@@ -141,13 +152,15 @@ def update_brief(brief_id):
...
+        sellers = brief.get('sellers')
         title = brief.get('title')
         return render_template_with_csrf(
             "view_buyers.html",
             users=users,
             title=title,
             brief_id=brief_id,
-            brief=brief
+            brief=brief,
+            sellers=sellers
```
Comment (ndavison) [inline app/main/views/buyers.py]: Do you need to bring this in any more? It seems like the `view_buyers.html` template is always referencing `brief.sellers` and not this param.

Comment (anthonymckerron) [reply app/main/views/buyers.py]: You're correct, I didn't need that anymore. I will remove it, cheers.

File: app/templates/view_buyers.html [hunk 4]
```diff
@@ -48,9 +48,13 @@
   {% endwith %}
 
   {% with heading = brief_id|string + ' - ' + title if title %}
-  {% include "toolkit/page-heading.html" %}
+    {% include "toolkit/page-heading.html" %}
   {% endwith %}
 
+{% if brief%}
```
Comment (tobiasbrown) [inline app/templates/view_buyers.html]:
```suggestion
{% if brief %}
```

File: app/templates/view_buyers.html [hunk 5]
```diff
@@ -98,18 +99,73 @@
...
+        %}
+            {% call summary.row() %}
+
+                {{ summary.field_name(brief.sellers[item].name.title()) }}
+
+                {% call summary.field() %}
+                    {{ item }}
+                {% endcall %}
+
+
```
Comment (tobiasbrown) [inline app/templates/view_buyers.html]:
```suggestion
```

File: app/templates/view_buyers.html [hunk 6]
```diff
@@ -98,18 +99,73 @@
...
+        <form action="{{ url_for('.update_brief',brief_id=brief_id) }}" method="post" class="question">
+            {{ form.csrf_token }}
+            <label class="question-heading" for="add_seller_to_rfx_brief">Add a seller to this RFx brief (seller code)</label>
+            <input type = "text" name="add_seller_to_rfx_brief" id="add_seller_to_rfx_brief" class="text-box"></input>
+            <br>
         <input type="submit" value="Save" class="button-save">
-    </form>
+        </form>
+    {% endif %}
+    {% if brief.lotSlug == 'rfx' %}
```
Comment (tobiasbrown) [inline app/templates/view_buyers.html]: Do we need this `if` block if we have the one defined above?

File: app/templates/view_buyers.html [hunk 7]
```diff
@@ -98,18 +99,73 @@
...
+    {% endif %}
+    {% if brief.lotSlug != 'rfx' %}
+        <form action="{{ url_for('.update_brief',brief_id=brief_id) }}" method="post" class="question">
+            {{ form.csrf_token }}
+            <label class="question-heading" for="edit_seller_email_list">Edit seller email list (List of sellers). One on each line</label>
+            <textarea rows="5" name="edit_seller_email_list" id="edit_seller_email_list" class="text-area">{{"\n".join (brief.sellerEmailList)}}</textarea>
+            <input type="submit" value="Save" class="button-save">
+        </form>
+    {% endif %}
+    {% if brief.lotSlug != 'rfx' %}
```
Comment (tobiasbrown) [inline app/templates/view_buyers.html]: Do we need this `if` block if we have the one defined above?

**Pull request merged**